### ISSUE TITLE
Some constants are still missing

### ISSUE THREAD

**Issue by villares** [berinhard/pyp5js#14]
Current workaround is to use `_P5_INSTANCE.` as a prefix to the original constant name.

Example:
```
HSB  # as in colorMode(HSB)
WEBGL  # as in createCanvas(500, 500, WEBGL)
```
And I have found where in p5.js all constants are defined :)
https://raw.githubusercontent.com/processing/p5.js/master/src/core/constants.js

**Comment by villares** [comment]
Hey @berinhard, this is a bit ugly, but I have added it to my static folder, referenced it in `index.html` and it kind of works:

https://gist.github.com/villares/4c48a7493c51431ca51931d72c205f50

**Comment by berinhard** [comment]
Thanks! Fixed by PR #15

**Comment by villares** [comment]
Sorry, this is not fixed yet.

It could be fixed by adding [constants.js](https://gist.github.com/villares/4c48a7493c51431ca51931d72c205f50) to assets and a proper reference in `index.html`.

There may be a better way of importing names in JS, directly from [p5js constants](https://raw.githubusercontent.com/processing/p5.js/master/src/core/constants.js) but I'm no sure how it might be done.
